### ISSUE TITLE
Rapification of macros fails if macro-strings have a certain length

### ISSUE THREAD

**Issue by Soldia1138** [KoffeinFlummi/armake#53]
This one gave me a headache. It seems that binarization/rapification sometimes fails if strings have a certain length.

I did some tests and this is the draft result:

```
#define MACRO_12 "0123456789012"
#define MACRO_13 "01234567890123"
#define MACRO_14 "012345678901234"
#define MACRO_15 "0123456789012345"
#define MACRO_16 "01234567890123456"
#define MACRO_17 "012345678901234567"
#define MACRO_18 "0123456789012345678"
#define MACRO_TEST "012345678"

arr12[] = {MACRO_12,MACRO_TEST,"Test","",""}; // works
arr13[] = {MACRO_13,MACRO_TEST,"Test","",""}; // does not work
arr14[] = {MACRO_14,MACRO_TEST,"Test","",""}; // does not work
arr15[] = {MACRO_15,MACRO_TEST,"Test","",""}; // works
arr16[] = {MACRO_16,MACRO_TEST,"Test","",""}; // works
arr17[] = {MACRO_17,MACRO_TEST,"Test","",""}; // works
arr18[] = {MACRO_18,MACRO_TEST,"Test","",""}; // works
```

This is true if the length of `MACRO_TEST` is >8. If it is <=8, everything's fine.

Smaller modifications resolve this issue (sometimes):
```
arr13[] = {MACRO_13,MACRO_TEST,"Test",""}; // works
arr14[] = {MACRO_14,MACRO_TEST,"Test",""}; // works
arr13[] = {MACRO_13,MACRO_TEST,"Test","","",""}; // works
arr14[] = {MACRO_14,MACRO_TEST,"Test","","",""}; // works
```

Deleting the third element `"Test"` helps as well.

I do not have a clue regarding the root cause...yet 😄

**Comment by Soldia1138** [comment]
What I forgot. The consequences :)

Armake simply ends in a dead-loop. Process in running forever @100% CPU

**Comment by KoffeinFlummi** [comment]
Fixed in #47.
